docs/upgrade.html

```diff
@@ -61,10 +67,8 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
             <a href="https://cwiki.apache.org/confluence/x/K5sODg">KIP-896</a> for the details.
         </li>
         <li>
-            Apache Kafka 4.0 only supports KRaft mode - ZooKeeper mode has been removed. As such, broker upgrades to 4.0.x (and higher) require KRaft mode and
-            the software and metadata versions must be at least 3.3.x (the first version when KRaft mode was deemed production ready). For clusters in KRaft mode
-            with versions older than 3.3.x, we recommend upgrading to 3.9.x before upgrading to 4.0.x. Clusters in ZooKeeper mode
-            have to be <a href="/39/documentation.html#kraft_zk_migration">migrated to KRaft mode</a> before they can be upgraded to 4.0.x.
+            Apache Kafka 4.0 only supports KRaft mode - ZooKeeper mode has been removed. About version upgrade,
+            check <a href="/{{version}}/documentation.html#upgrade_4_0_0">Upgrading to 4.0.0 from any version 3.3.x through 3.9.x</a> for more info.
```

Comment:
Changing 4.0.0 to 4.0.x would make document more consistent.

Reply:
Sounds good. PR updated. Thanks.